lib/forge2d_game.dart

```diff
@@ -30,6 +30,7 @@ class Forge2DGame extends BaseGame {
     world = World(gravity);
     world.setContactListener(_contactCallbacks);
     viewport = Viewport(viewportSize, scale);
+    viewport.yFlip = true;
```

Comment:
Nit:

```dart
viewport = Viewport(viewportSize, scale)
  ..yFlip = true;
```

Reply:
Ah line shouldn't be in here at all, good that you commented on it.
I was just testing things.

Reply:
😎